src/components/forms/OrganizationSelect/index.tsx

```diff
@@ -1,5 +1,5 @@
 import reducerRegistry from '@onaio/redux-reducer-registry';
-import React, { useEffect } from 'react';
```

Comment:
I do not think we need any changes to this component.  Can you please revert these?